packages/core/src/components/tabs/tabPanel.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2024 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import classNames from "classnames";
+import * as React from "react";
+
+import { AbstractPureComponent, Classes, Utils } from "../../common";
+
+import { type TabProps } from "./tab";
+import { generateTabIds, type TabTitleProps } from "./tabTitle";
+
+export interface TabPanelProps extends Pick<TabProps, "className" | "id" | "panel">, Pick<TabTitleProps, "parentId"> {
+    /**
+     * Used for setting `aria-hidden` prop.
+     */
```

Comment:
```suggestion
    /**
     * If true, will set required accessibility roles and set `display` to `none`.
     */
```

Reply:
I'm thinking maybe we just make this `required` if the expected usage is that one `TabPanel` is provided for each tab, like is done in the uncontrolled usage of `Tabs`. In your example you essentially share a `TabPanel` but I'm wondering if making this optional encourages that pattern.

Reply:
sure